packages/synapse-react-client/src/lib/containers/SynapseNavDrawer.tsx

```diff
@@ -123,18 +123,16 @@ const NavDrawerListItem = (props: MenuItemParams) => {
 
   const listItem = (
     <Tooltip title={tooltip} placement="right">
-      <ListItem
-        button
+      <ListItemButton
```

Comment:
ListItem with `button` prop is deprecated. The ListItemButton component is the preferred replacement